v2/internal/controllers/crd_dbformysql_flexibleserver_test.go

```diff
@@ -205,7 +205,7 @@ func MySQLFlexibleServer_AADAdmin_CRUD(tc *testcommon.KubePerTestContext, rg *re
 		Spec: mysqlpreview.FlexibleServers_Administrator_Spec{
 			Owner:             testcommon.AsOwner(server),
 			AdministratorType: &aadAdmin,
-			Login:             &mi.Name,
+			Login:             &mi.Name, // TODO: Should we be able to source this from a configmap?
```

Comment:
Remove this TODO, I think for now it doesn't make sense to source from configMap. We can revisit if user asks